File: android/Arquitetura Android/TwitchAppRetrofitExample/app/src/main/java/br/com/adalbertofjr/twitchappretrofitexample/MainActivity.java
package br.com.adalbertofjr.twitchappretrofitexample;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.util.Log;

import java.util.List;

import javax.inject.Inject;

import br.com.adalbertofjr.twitchappretrofitexample.http.TwitchAPI;
import br.com.adalbertofjr.twitchappretrofitexample.http.apimodel.DataGame;
import br.com.adalbertofjr.twitchappretrofitexample.http.apimodel.Twitch;
import br.com.adalbertofjr.twitchappretrofitexample.root.App;
import retrofit2.Call;
import retrofit2.Callback;
import retrofit2.Response;

public class MainActivity extends AppCompatActivity {

    @Inject
    TwitchAPI twitchAPI;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        ((App) getApplication()).getComponent().inject(this);

        Call<Twitch> twitchCall = twitchAPI.getTopGames("key_twitch_api");

        twitchCall.enqueue(new Callback<Twitch>() {
            @Override
            public void onResponse(Call<Twitch> call, Response<Twitch> response) {
                List<DataGame> topGames = response.body().getData();

                for (DataGame game : topGames) {
                    Log.i("Jogo: ", game.getName());
                }
            }

            @Override
            public void onFailure(Call<Twitch> call, Throwable t) {
                t.printStackTrace();
            }
        });


    }
}

File: android/CarouselApp/app/src/main/java/br/com/adalbertofjr/carouselapp/MainActivity.kt
package br.com.adalbertofjr.carouselapp

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import br.com.adalbertofjr.carouselapp.view.carousel.adapter.CarouselViewPagerAdapter
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
    private val items = listOf("Item 1", "Item 2", "Item 3", "Item 4")

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        carousel.adapter = CarouselViewPagerAdapter(this, items)
    }
}
